whelk-core/src/main/groovy/whelk/component/PostgreSQLComponent.groovy

```diff
@@ -2290,6 +2292,29 @@ class PostgreSQLComponent {
         }
     }
 
+    List<DocumentVersion> loadDocumentHistory(String id) {
+        return withDbConnection {
+            Connection connection = getMyConnection()
+            PreparedStatement selectstmt = null
+            ResultSet rs = null
+            List<DocumentVersion> docList = []
+            try {
+                selectstmt = connection.prepareStatement(GET_ALL_DOCUMENT_VERSIONS)
+                selectstmt.setString(1, id)
+                rs = selectstmt.executeQuery()
+                int v = 0
+                while (rs.next()) {
+                    def doc = assembleDocument(rs)
+                    doc.version = v++
```

Comment:
Never used as far as I can tell, but it numbering is backwards?
v 0 == current
Or is that intentional?

Reply:
Ok, same numbering as in `doLoadAllVersions`

Reply:
> Ok, same numbering as in `doLoadAllVersions`

Yeah, its used here (where that needs to be the order):
java/whelk/export/marc/ProfileExport.java:174